pkg/apis/application/v1alpha1/types.go

```diff
@@ -191,6 +191,8 @@ type ApplicationSource struct {
 	Chart string `json:"chart,omitempty" protobuf:"bytes,12,opt,name=chart"`
 	// Ref is reference to another source within sources field. This field will not be used if used with a `source` tag.
 	Ref string `json:"ref,omitempty" protobuf:"bytes,13,opt,name=ref"`
+	// Name is used to refer to a source and is displayed in the UI.
```

Comment:
Considering we want this to be of use only for a multi-source app, can we update the comment to reflect the same?

Reply:
Sure thing. Updated.